### ISSUE TITLE
Manual skip option

### ISSUE THREAD

**Issue by knoellle** [polymorphicshade/NewPipe#37]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

In the browser extension, I can select from one of four options for each segment type:

- Disable
- Show in seek bar
- Manual skip
- Auto skip

In the app, this is just a toggle between disable and auto skip.
It would be lovely to get at least one of the two missing modes in the app as well.

#### Is your feature request related to a problem? Please describe it

In the current state, it is only possible to auto-skip or to disable skipping and highlighting in the seek bar completely.
Even when manually seeking to a point inside a segment that is set to be skipped, it automatically jumps to the end.
AFAIK it is currently not possible to watch any skipped segments unless I change the options. 

#### Additional context

The show in seek bar option should be pretty straight forward.
For the manual skipping, a more compact version of the way it's done in the browser extension would be cool.
![image](https://user-images.githubusercontent.com/33729490/109153576-62f07b00-776d-11eb-80f3-6ab2d22572ef.png)

Not sure how a manual skip button would work in the smaller popup player.

#### How will you/everyone benefit from this feature?

My preferred mode of using it is to have all types on manual so I can listen to the start of the sponsor segments and then decide whether it's just a canned clip that I've already seen a bunch of times or whether it's an actually creative ad (Internet Historian comes to mind).
As is, the only way to view a segment where you disagree with whoever placed it is to change the options or use another app.

**Comment by polymorphicshade** [comment]
I've been meaning to implement this kind of thing... thanks for the reminder! I only use auto-skip on my browser so I usually forget there are other modes in SponsorBlock 😅

**Comment by Stenledtrap** [comment]
> I've been meaning to implement this kind of thing... thanks for the reminder! I only use auto-skip on my browser so I usually forget there are other modes in SponsorBlock 😅

I didn't know where else to ask you that, but can i submit my own segments or is it not a feature (yet).

**Comment by polymorphicshade** [comment]
@Stenledtrap I will not implement the ability to submit segments because I want to try to keep NewPipe's original intent by not associating any user activity with videos.

**Comment by Stenledtrap** [comment]
> @Stenledtrap I will not implement the ability to submit segments because I want to try to keep NewPipe's original intent by not associating any user activity with videos.

I see. Where are the submissions coming from? You don't do them yourself right?

**Comment by knoellle** [comment]
The submissions are from this project: https://github.com/ajayyy/SponsorBlock/

**Comment by pokegamer5547** [comment]
@polymorphicshade ah sorry I know this might not be the place to ask this but I couldn't create issue with question label or without label.  I noticed in sponsor block extension and even in YouTube vanced that there is a option which will send the information to sponserblock site each time segment is skipped and its an optional toggle. So does newpipe x sponserblock also sends information to sponserblock site each time segment is skipped?

**Comment by polymorphicshade** [comment]
@pokegamer5547 nope. The automatic skipping is completely client-side - I don't send anything to the SponsorBlock host API when the code skips a video segment.

**Comment by pokegamer5547** [comment]
> @pokegamer5547 nope. The automatic skipping is completely client-side - I don't send anything to the SponsorBlock host API when the code skips a video segment.

Oh I see, that's a good thing. Thanks for telling me :)

**Comment by castrik** [comment]
while at it, also look at adding an unskip button (go back to see the segment, if you accidentally skipped it)

**Comment by pkillnine** [comment]
This could be done the same way Netflix does, have a small box in the bottom right corner overlaid on the video saying "Skip intro", but replace "intro" with whatever sponsorblock category is currently playing.

![image](https://imgur.com/NkbcH1v.png)

**Comment by Relected** [comment]
I'd like to bump this issue because I was used to YouTube Vanced and how they handle sponserblock better

it would be VERY cool to be able to manually skip each segment we don't want manually the same way how YouTube Vanced does it

**Comment by onnyyonn** [comment]
I am bumping this issue because Youtube Vanced just got discontinued. Newpipe will get a lot of new users who are used to this feature.

**Comment by Malotun** [comment]
I hope this gets added, it is nice to have the option.

**Comment by DylwinTFTW** [comment]
Also hope this gets added.

**Comment by ArijitBh** [comment]
Hoping this gets added

**Comment by Atemu** [comment]
If you want this to be added, use the reactions on the main post rather than spamming people's inboxes, thanks.
